core/lib/data/utils/compress_file_utils.dart

```diff
@@ -0,0 +1,40 @@
+
+import 'dart:typed_data';
+
+import 'package:core/utils/app_logger.dart';
+import 'package:core/utils/build_utils.dart';
+import 'package:flutter_image_compress/flutter_image_compress.dart';
+
+class CompressFileUtils {
+
+  /// Maximum image size displayed (Unit: KB)
+  static const int MAXIMUM_IMAGE_SIZE = 300;
+  /// Quality image 0 - 100
+  static const int QUALITY_DEFAULT = 50;
+
+  bool _exceedMaximumImageSize(Uint8List bytesData) {
+    final maximumSizeBytes = MAXIMUM_IMAGE_SIZE * 1024;
+    final sizeBytesData = bytesData.length;
+    log('DownloadClient::exceedMaximumImageSize(): maximumSizeBytes: $maximumSizeBytes');
```

Comment:
please carefully about the log, should be exactly class name for easier in debug. Please use `live templeate`.

Reply:
It's just a copy error.

Reply:
please use `live template` to avoid that. please! please!